### DISCUSSION TITLE
Customising Github Release artifacts

### DISCUSSION THREAD

**Discussion by dimitrisnl** [changesets/changesets#1086] [Q&A]
Hello!

I'm working with a private package, that doesn't need to be uploaded to the NPM. I need the whole Changeset flow (versioning, bundled changes in a PR), except for pushing to NPM.

My only requirement is creating a GitHub Release, with a zip file of the (uncommitted) `/dist` folder.

I've combed through the docs, but I'm unsure how to achieve this. Can you point me in the right direction?

**Comment by dimitrisnl** [comment]
There's this entry of [custom publishing on the action](https://github.com/changesets/action#custom-publishing) but there's an already generated GitHub release with the source files attached. 

I've also thought about using [Action: Upload files to a GitHub release](https://github.com/marketplace/actions/upload-files-to-a-github-release)

**Reply by Dwarf1er** [reply]
I was curious to know what you ended up doing about this? 

I've been having the same questions since I'm also releasing other non-npm applications as part of my monorepo and I've been having a hard time adding binaries or other artifacts to the release created by the changesets action. Coordinating releases to happen when merging the release PR instead of the initial PR to the main branch has been a bit messy on my end too I have to say 🤔 

Cheers
